configurator/src/main.rs

```diff
@@ -80,12 +80,25 @@ fn capturer_from_options(
     };
 
     let light_client: Box<dyn light_client::LightClient> = if let Some(endpoint) = lights_endpoint {
-        if endpoint.starts_with("http://") {
-            info!("Using remote HTTP light client at endpoint: {}", endpoint);
-            Box::new(light_client::RemoteLightClient::new(&endpoint))
-        } else {
-            info!("Using local TTY light client");
-            Box::new(light_client::tty::TtyLightClient::new()?)
+        match endpoint {
```

Comment:
I'm not sure if the match version is more readable in this case. Up to you though

Reply:
Yeah, I'm not a huge fan of it either, but IMO it's easier on the eyes than the sequence of `else if`'s